source/common/common/utility.h

```diff
@@ -68,8 +68,9 @@ class DateFormatter {
     const size_t width_;
 
     // The string before the current specifier's position and after the previous found specifier. A
-    // segment may include strftime accepted specifiers. E.g. given "%3f-this-i%s-a-segment-%4f",
-    // the current specifier is "%4f" and the segment is "-this-i%s-a-segment-".
+    // segment may include absl::FormatTime accepted specifiers. E.g. given
```

Comment:
I think we might need documentation updates here? Unfortunately I think this might be a user visible breaking change since I think we accept time format strings in access log configuration? I'm not sure if we can actually do a breaking change here and we might need to convert any differences between the two? :(

Reply:
From the comment above absl::FormatTime, it looks like it accepts all the strftime specifiers with some additions:
https://github.com/abseil/abseil-cpp/blob/b312c3cb53a0aad75a85ac2bf57c4a614fbd48d4/absl/time/time.h#L1197-L1233

If we track down the implementation, it looks like strftime is being used under the hood:
https://github.com/abseil/abseil-cpp/blob/b312c3cb53a0aad75a85ac2bf57c4a614fbd48d4/absl/time/format.cc#L70-L76
https://github.com/google/cctz/blob/6ad4fee521f8cfe1d3374b9422907c31e48c641e/src/time_zone_format.cc#L271-L292

Therefore, it shouldn't be a breaking change.

Reply:
I notice you changed the format string here: https://github.com/envoyproxy/envoy/pull/6029/files#diff-4c9c8a857896d3154dddfa1ae5934059R411. Is that just because it can do the millseconds timing by default? Either way I think we can then change the docs here https://www.envoyproxy.io/docs/envoy/latest/configuration/access_log#command-operators to link to the absl docs? Or do we just want to say that we support strftime only?

Reply:
Yes - we changed the format string just because absl::FormatTime can handle the milliseconds by default.
Regarding the documentation, we don't have a strong opinion.

Reply:
OK sounds good we can leave it as is for now. Thank you for explaining!